### DISCUSSION TITLE
有没有交流群什么的？

### DISCUSSION THREAD

**Discussion by tidexe** [yjl9903/AnimeGarden#1411] [General]
有没有交流群什么的？现在好像是崩了？
国内是dns劫持了，美国硅谷和日本节点都返回Connection reset

**Comment by yjl9903** [comment]
昨天因为某些原因全量切换到了另一个域名，仓库内的相关的链接都已经更新。

**Reply by yjl9903** [reply]
没有群，但可以现开一个 https://t.me/+BDEzp1bAtbwxZjA9
